dbms/programs/server/MySQLHandler.h

```diff
@@ -12,6 +12,8 @@
 namespace DB
 {
 
+
```

Comment:
Too many empty lines (code style limit is 2 empty lines, but one line seems to be enough here)

Reply:
fixed